bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/handler/ShellyColorUtils.java

```diff
@@ -29,10 +28,34 @@
  */
 @NonNullByDefault
 public class ShellyColorUtils {
+    OnOffType power = OnOffType.OFF;
+    String mode = "";
+
+    int red = 0;
+    int green = 0;
+    int blue = 0;
+    int white = 0;
+    PercentType percentRed = new PercentType(0);
+    PercentType percentGreen = new PercentType(0);
+    PercentType percentBlue = new PercentType(0);
+    PercentType percentWhite = new PercentType(0);
+
+    int gain = 0;
+    int brightness = 0;
+    int temp = 0;
+    int minTemp = 0;
+    int maxTemp = 0;
+    PercentType percentGain = new PercentType(0);
+    PercentType percentBrightness = new PercentType(0);
+    PercentType percentTemp = new PercentType(0);
+    Integer effect = 0;
```

Comment:
Why are all these fields package private?

Reply:
No, they are accessed in ShellyLightHandler